templates/default/marathon.yaml.erb

```diff
@@ -0,0 +1,8 @@
+# Generated by Chef, local modifications will be overwritten
+
+<%= JSON.parse(({ 'instances' => @instances }).to_json).to_yaml %>
+
+init_config:
```

Comment:
How about this approach?
https://github.com/DataDog/chef-datadog/blob/10a9bfcf03e98ff2bff9f34b7321dee9cd29e2c6/templates/default/docker_daemon.yaml.erb